VoxelisedSimulation/SubScripts/EstimateRandomsFromDelayed.sh

```diff
@@ -0,0 +1,40 @@
+## #! /bin/sh
+
+## AUTHOR: Robert Twyman
+## AUTHOR: Kris Thielemans
+## Copyright (C) 2020 University College London
+## Licensed under the Apache License, Version 2.0
+
+## This script estimates the background randoms from the unlisted (Trues+Randoms+Scattered) coincidences and the measured Delayed coincidences.
+
+## PARAMETERS
+
+if [ $# != 3 ]; then
+	echo "Usage: EstimateGATESTIRNorm.sh OutputFilename MeasuredData (Coincidences) DelayedData"
+	exit 1
+fi 
+
+OutputFilename=$1 
+MeasuredData=$2  ## Trues+Randoms+Scattered Coincidences Sinogram
+DelayedData=$3 ## Delayed Coincidences Sinogram
+
+factors=randoms_factors
+
+## ML Normfactors loop numbers (Hardcoded for now)
+outer_iters=5
+eff_iters=6
+
+## find ML normfactors
+echo "find_ML_normfactors3D"
+find_ML_normfactors3D $factors $MeasuredData $DelayedData $outer_iters $eff_iters
```

Comment:
This is wrong. The command line is
```
find_ML_normfactors3D out_filename_prefix measured_data model num_iterations num_eff_iterations
```
To estimate singles, the `measured_data` = `$DelayedData` and the `model` is just `ones`. And you don't want geometric factors (although I guess they don't do any harm, it's just that the model is `2 tau singles1 singles2 duration_factor`, no geo-factors there. So really you should set `out_iters` to 0 to do only "efficiencies"

Reply:
>This is wrong. The command line is
find_ML_normfactors3D out_filename_prefix measured_data model num_iterations num_eff_iterations

This is delayed though, not singles. The delayed coincidences are random (or random scattered) coincidences. 

> So really you should set out_iters to 0 to do only "efficiencies"

Ill change that.

Reply:
> This is delayed though, not singles. The delayed coincidences are random (or random scattered) coincidences.

yup, but you estimate singles from delayeds. so the data has to be the delayeds. the output are the singles (in this case, using the "efficiencies" because they have the same model)....

Reply:
Sorry I misinterpreted what I was doing. Got it now.